cloud/aws/stress/supervisor.conf

```diff
@@ -0,0 +1,29 @@
+[inet_http_server]
+port=*:9001
+
+[supervisord]
+logfile=%(here)s/logs/supervisor.log
+pidfile=%(here)s/supervisor.pid
+childlogdir=%(here)s/logs
+directory=%(here)s
+
+; the below section must remain in the config file for RPC
+; (supervisorctl/web interface) to work, additional interfaces may be
+; added by defining them in separate rpcinterface: sections
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl=http://127.0.0.1:9001 ; use an http:// url to specify an inet socket
+
+[program:stress]
+directory=%(here)s
+command=%(here)s/stress.sh
+process_name=%(program_name)s
+numprocs=1
+autostart=false
+autorestart=false
```

Comment:
I mostly use it to monitor that all processes are still running (this is done in the cockroach-prod script). Very useful, especially when there's more than one node.